packages/core/components/Modal/CopyFileManifest/index.tsx

```diff
@@ -92,13 +89,20 @@ export default function CopyFileManifest({ onDismiss }: ModalProps) {
     );
 
     const [fileDetails, setFileDetails] = React.useState<FileDetail[]>([]);
+    const [loading, setLoading] = React.useState<boolean>(true);
 
     React.useEffect(() => {
-        async function fetchDetails() {
-            const details = await fileSelection.fetchAllDetails();
-            setFileDetails(details);
+        async function fetchFiles() {
+            try {
+                const details = await fileSelection.fetchAllDetails();
+                setFileDetails(details);
+            } catch (err: any) {
+                throw new Error(err.message || "Error fetching file details.");
```

Comment:
This error won't get seen by a user in a useful manner. Having it be set to the component's state like you had earlier and then displaying the error would make this more useful

Reply:
Resolved in [c69c4c8](https://github.com/AllenInstitute/biofile-finder/pull/411/commits/c69c4c89f925a7b94f5fbcd946d82463bf23d721)